docs/data/data-grid/demo/PopularFeaturesDemo.tsx

```diff
@@ -0,0 +1,349 @@
+    case 'pro':
+      return { avatarLink: '/static/x/pro.svg', color: '#c8e9ff' };
+    default:
+      return { avatarLink: undefined, color: '#c8ffdb' };
+  }
+};
+
+const PlanTag = (props: { plan: string }) => {
+  const chipPropperties = getChipProperties(props.plan);
+  const avatar = !chipPropperties.avatarLink ? undefined : (
+    <Avatar src={chipPropperties.avatarLink} />
+  );
+  return (
+    <Chip
+      avatar={avatar}
+      sx={{ background: chipPropperties.color, color: 'rgba(0, 0, 0, 0.87)' }}
+      label={props.plan}
+    />
+  );
+};
+
+const CustomToolbar = () => {
+  return (
+    <GridToolbarContainer sx={{ p: 1 }}>
+      <GridToolbarQuickFilter />
+    </GridToolbarContainer>
+  );
+};
+
+const RowDemo = (props: any) => {
+  const { row } = props;
+  const theme = useTheme();
+  const gridBgColor = theme.palette.mode === 'dark' ? '#000' : '#fff';
+  const panelColor = theme.palette.mode === 'dark' ? 'transparent' : '#efefef';
+
+  return (
+    <Box sx={{ py: 2, background: panelColor }}>
+      <Box style={{ width: '90%', margin: 'auto', background: gridBgColor }}>
+        {row.demo}
+      </Box>
+    </Box>
+  );
+};
+
+const columns: GridColDef[] = [
+  {
+    field: 'name',
+    headerName: 'Feature name',
+    maxWidth: 160,
+    flex: 0.2,
+    minWidth: 100,
+    groupable: false,
+    renderCell: (params) => {
+      if (!params.value) {
+        return <React.Fragment />;
+      }
+      return (
+        <Box sx={{ width: '100%', height: '100%', position: 'relative' }}>
+          <Typography sx={{ fontSize: '1rem', fontWeight: '500' }}>
+            <Link
+              href={`/x/react-data-grid${params.row.detailPage}`}
+              target="_blank"
+            >
+              {params.value}
+            </Link>
+          </Typography>
+          {params.row.newBadge && (
+            <Box
+              sx={{
+                width: 'fit-content',
+                height: 'fit-content',
+                fontSize: '0.8em',
+                fontWeight: 600,
+                position: 'absolute',
+                textAlign: 'center',
+                top: -13,
+                left: -20,
+                background: '#fcf0a0',
+                color: '#af5b00',
+                px: 1,
+                borderRadius: 10,
+              }}
+            >
+              New
+            </Box>
+          )}
+        </Box>
+      );
+    },
+  },
+  {
+    field: 'description',
+    headerName: 'Brief description',
+    groupable: false,
+    flex: 0.5,
+    minWidth: 120,
+  },
+  {
+    field: 'plan',
+    headerName: 'Plan',
+    maxWidth: 130,
+    flex: 0.3,
+    type: 'singleSelect',
+    valueOptions: ['Premium', 'Pro', 'Community'],
+    renderCell: (params: GridRenderCellParams<string>) => {
+      if (!params.value) {
+        return <React.Fragment />;
+      }
+      return <PlanTag plan={params.value} />;
+    },
+  },
+];
+
+function PopularFeaturesDemo() {
```

Comment:
AFAIK the convention with the demos:

```suggestion
export default function PopularFeaturesDemo() {
```